.idea/misc.xml

```diff
@@ -12,5 +12,11 @@
     </writeAnnotations>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="FrameworkDetectionExcludesConfiguration">
+    <file type="web" url="file://$PROJECT_DIR$" />
+  </component>
+  <component name="JavaScriptSettings">
```

Comment:
@keturn we'll see more and more of these unrelated changes, right? Ultimate vs Community edition, as well as plugins installed in addition to the recommended ones will add to these files if I get it right... 🙄